src/archive.rs

```diff
@@ -291,24 +182,53 @@ impl<'a> Archive<Read + 'a> {
         }
         Ok(())
     }
+}
+
+impl<'a, R: Read> Entries<'a, R> {
+    /// Indicates whether this iterator will return raw entries or not.
+    ///
+    /// If the raw list of entries are returned, then no preprocessing happens
+    /// on account of this library, for example taking into accout GNU long name
+    /// or long link archive members. Raw iteration is disabled by default.
+    pub fn raw(self, raw: bool) -> Entries<'a, R> {
```

Comment:
This method is misleading in my opinion. I'd have called it `into_<something>` as it takes `self` by value, and maybe split it in two functions like `into_raw` and `into_auto`. Another possibility is to use a function like `pub fn set_raw(&mut self, raw: bool)`.

Another thing: the documentation says "Indicates whether this iterator will return raw entries or not", but it rather turns the iterator into something else. Only a thought here. The API itself is good to have!